app/models/concerns/turbo/broadcastable.rb

```diff
@@ -112,10 +135,34 @@ def broadcasts(stream = model_name.plural, inserts_by: :append, target: broadcas
       after_destroy_commit -> { broadcast_remove }
     end
 
+    # Configures the model to broadcast a "page refresh" on creates, updates, and destroys to a stream
+    # name derived at runtime by the <tt>stream</tt> symbol invocation.
+    def broadcasts_refreshes_to(stream)
+      after_commit -> { broadcast_refresh_later_to(stream.try(:call, self) || send(stream)) }
+    end
+
+    # Same as <tt>#broadcasts_refreshes_to</tt>, but the designated stream for page refreshes is automatically set to
+    # the current model.
+    def broadcasts_refreshes
+      after_commit -> { broadcast_refresh_later }
```

Comment:
So, this will only send broadcast refreshes to the current model's channel, right? Shouldn't it be closer to the broadcasts method and accept the stream argument or default to the model's plural name when creating the model? Otherwise, it will send a broadcast to a channel that no one will ever be listening on (because the model is being created), right?
```suggestion
    def broadcasts_refreshes(stream = model_name.plural)
      after_create_commit -> { broadcast_refresh_later_to(stream) }
      after_update_commit -> { broadcast_refresh_later }
```

Reply:
Sent a PR: https://github.com/hotwired/turbo-rails/pull/521